daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/validation/ValidationSpec.scala

```diff
@@ -354,57 +338,39 @@ class ValidationSpec extends AnyFreeSpec {
 
   //--[Fetch node tweaks]--
 
-  private val tweakFetchCoid = Tweak.single[Node] {
-    case nf: Node.NodeFetch[_] => Some(nf.copy(coid = changeContractId(nf.coid)))
-    case _ => None
+  private val tweakFetchCoid = Tweak.single[Node] { case nf: Node.NodeFetch[_] =>
+    nf.copy(coid = changeContractId(nf.coid))
   }
-  private val tweakFetchTemplateId = Tweak.single[Node] {
-    case nf: Node.NodeFetch[_] => Some(nf.copy(templateId = changeTemplateId(nf.templateId)))
-    case _ => None
+  private val tweakFetchTemplateId = Tweak.single[Node] { case nf: Node.NodeFetch[_] =>
+    nf.copy(templateId = changeTemplateId(nf.templateId))
   }
-  private val tweakFetchOptLocation = Tweak[Node] {
-    case nf: Node.NodeFetch[_] =>
-      tweakOptLocation.run(nf.optLocation).map { x => nf.copy(optLocation = x) }
-    case _ => Nil
+  private val tweakFetchOptLocation = Tweak[Node] { case nf: Node.NodeFetch[_] =>
+    tweakOptLocation.run(nf.optLocation).map { x => nf.copy(optLocation = x) }
   }
-  private val tweakFetchActingPartiesEmpty = Tweak[Node] { //insig
-    case nf: Node.NodeFetch[_] =>
-      if (nf.actingParties.isEmpty) {
-        tweakPartySet.run(nf.actingParties).map { x => nf.copy(actingParties = x) }
-      } else Nil
-    case _ => Nil
+  private val tweakFetchActingPartiesEmpty = Tweak[Node] {
+    case nf: Node.NodeFetch[_] if (nf.actingParties.isEmpty) => //insig
```

Comment:
```suggestion
    case nf: Node.NodeFetch[_] if nf.actingParties.isEmpty => //insig
```
parenthesis are not required in guard

Reply:
done